web-gateway/src/main/resources/application.conf

```diff
@@ -1,5 +1,5 @@
 play.http.secret.key = "somesecret"
 online-auction.instruction.show = true
-
+lagom.persistence.read-side.global-prepare-timeout = 60s
```

Comment:
If this doesn't fix the problem for you, then it should probably be removed. We'll need to find some other solution.